src/dotnet/upack/Version.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.ComponentModel;
+using System.Diagnostics;
+using System.Reflection;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Inedo.UPack.CLI
+{
+    [DisplayName("version")]
+    [Description("Outputs the installed version of upack.")]
+    public class Version : Command
+    {
+        public override Task<int> RunAsync(CancellationToken cancellationToken)
+        {
+            var assembly = Assembly.GetExecutingAssembly();
+            var fvi = FileVersionInfo.GetVersionInfo(assembly.Location);
```

Comment:
You can get the version without reloading the file by calling `GetName()` on the assembly.